reindexer/terraform/outputs.tf

```diff
@@ -10,3 +10,7 @@ output "topic_arn" {
 output "mets_reindexer_topic_name" {
   value = "${local.mets_reindexer_topic_name}"
 }
+
+output "mets_reindexer_topic_arn" {
+  value = "${local.mets_reindexer_topic_arn}"
```

Comment:
```suggestion
  value = local.mets_reindexer_topic_arn
```

Reply:
This is in the reindexer which is still tf0.11